kong/db/migrations/core/001_14_to_15.lua

```diff
@@ -33,6 +33,10 @@ return {
       $$;
 
       ALTER TABLE IF EXISTS ONLY "plugins" ALTER "config" TYPE JSONB USING "config"::JSONB;
+
+
+
+      ALTER TABLE IF EXISTS ONLY "upstreams" ALTER "healthchecks" TYPE JSONB USING "healthchecks"::JSONB;
```

Comment:
Out of curiosity: do you know what happens _if_ (big if) the type of this column is already `JSONB`?

Reply:
I am not sure what happens internally, but I guess pretty much nothing happens. I tried it and, well nothing happened.